.gitignore

```diff
@@ -1,5 +1,4 @@
 # IDE and editor
-.vscode
```

Comment:
Will the `.vscode` folder contain other files? Do we only need to include `.vscode/settings.json`?